test/unittest/tests/wrappers/mock_session.cpp

```diff
@@ -34,3 +41,18 @@ MockSession::buildTestMockSession()
     // Construct an object that will now own the two pointers passed in.
     return std::shared_ptr<MockSession>(new MockSession(sessionImpl, mockConnection));
 }
+
+int
+handleWiredTigerError(
+  WT_EVENT_HANDLER *handler, WT_SESSION *session, int error, const char *message)
```

Comment:
In C++ (and unlike C), we can comment out or completely remove unused parameters.

For example the following will eliminate unused parameters warnings/errors:

`  WT_EVENT_HANDLER *handler, WT_SESSION */*session*/, int /*error*/, const char *message) `

Reply:
In that case I think we can just leave them empty, I will do that.